managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -124,6 +124,17 @@ public UniverseController(YBClientService service) {
     this.ybService = service;
   }
 
+  private boolean validateEncryption(ObjectNode formData) {
```

Comment:
Hm... Before this I saw two ways to implement validate functions - without result which throws exceptions if something is wrong, another - which returns true if the arguments is successfully validated. Your implementation can confuse someone as it implements inverted logic - it returns `true` if something is wrong. Actually I prefer standard conventions.

Reply:
ok got it

Reply:
Done